templates/index.html

```diff
@@ -76,23 +76,16 @@ <h1>NOTAM Sorter</h1>
 	function addDestination() {
 		var additionalDestinationsDiv = document.getElementById('additionalDestinations');
 
-		// Create a new destination row if there are no rows or the last row is full
-		if (!additionalDestinationsDiv.lastChild || additionalDestinationsDiv.lastChild.childElementCount === 2) {
-			var destinationRow = document.createElement('div');
-			destinationRow.classList.add('row');
-        		additionalDestinationsDiv.appendChild(destinationRow);
-		}
+		// Create a new destination row
+		var destinationRow = document.createElement('div');
+		destinationRow.classList.add('row');
+		var destinationInput = document.createElement('div');
+		destinationInput.classList.add('col-md-6', 'mb-3');
+		destinationInput.innerHTML = '<label for="destAirport' + destinationCounter + '" class="form-label">Additional Destination ' + destinationCounter + '</label>' +
+			'<input type="text" class="form-control" name="destAirport' + destinationCounter + '" placeholder="Airport Code" autocomplete="off">';
+		destinationRow.appendChild(destinationInput);
+		additionalDestinationsDiv.appendChild(destinationRow);
 
-		// Create Destination Location input
```

Comment:
The second way is better because it adds new information without wasting space, by putting it in the last spot available. so yeah I mean to change it. It works great.